tcf_website/views/search.py

```diff
@@ -0,0 +1,106 @@
+"""Views for search results"""
+import os
+import json
+import requests
+
+from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+
+@login_required
```

Comment:
Maybe no `@login_required`? Unless we do want to make users log in to search?

Reply:
I think we probably should. Do we make people login to see browse?

Reply:
Currently we're letting people browse when not logged in

Reply:
I think we probably should make people login so they are forced to make an account and thus can write reviews. Otherwise people probably wouldn't take the time to make an account just to write reviews and would treat the website as a read-only thing

Reply:
We're using Google auth so making an account wouldn't have that much friction-- does it make sense to have browse be login-not-required and search login-required? Should we make both login-required or both login-not-required?

Reply:
@j-alicia-long @saikonuri thoughts?

Reply:
I personally think we should make both login-required

Reply:
How about for consistency, we remove it for now and then discuss with the team on Monday?

Reply:
Sounds good!

Reply:
Sounds good to me as well. I do think we should have login for both but yeah we can discuss on monday